packages/dotcom-server-app-context/src/__test__/validate.spec.ts

```diff
@@ -13,7 +13,11 @@ describe('dotcom-server-app-context/src/validate', () => {
 
   it('throws an error when given an unknown property/value', () => {
     expect(() => subject('thisProperty', 'isNotInTheSchema')).toThrow(
-      'Validation error: data should NOT have additional properties, received "isNotInTheSchema"'
+      `
+Validation error: data should NOT have additional properties, received "isNotInTheSchema"
+
+If you want to share application specific data with the client, consider using @financial-times/dotcom-ui-data-embed.
+`.trim()
```

Comment:
Backticks can be frustrating. I wonder if there's a way to do multiline without inconsistent formatting and `trim()`… 🤔

Reply:
We could consider adding a dependency like https://www.npmjs.com/package/outdent but I felt like that was a bit much for this small usecase. What do you think?

Reply:
We could always just do something like

```javascript
'string\n' +
'second-string` 
```

Reply:
Yeah, I agree with your feeling about importing a package just for this.

You could even have one really long line…

None is particularly nice so I will leave it to you to decide (which may as well be how it is currently).